packages/plugin-network-security-config/src/index.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Defines a plugin for @brandingbrand/code-cli-kit.
+ * @module Plugin
+ */
+
+import {
+  NetworkSecurityConfigXML,
+  definePlugin,
+  fs,
+  path,
+  withNetworkSecurityConfig,
+} from '@brandingbrand/code-cli-kit';
+
+import type {
+  CodePluginNetworkSecurityConfig,
+  NetworkSecurityConfigKind,
+} from './types';
+
+const baseConfigs: Record<
+  NetworkSecurityConfigKind,
+  NetworkSecurityConfigXML['network-security-config']
+> = {
+  debug: {
+    'debug-overrides': {
+      'trust-anchors': {
+        certificates: [{$: {src: 'user'}}, {$: {src: 'system'}}],
```

Comment:
do we need these debug overrides if we are just setting them in the base config?